library/std/src/macros.rs

```diff
@@ -313,14 +313,36 @@ macro_rules! dbg {
         // of temporaries - https://stackoverflow.com/a/48732525/1063961
         match $val {
             tmp => {
+                let val = $crate::stringify!($val);
+                // Remove the invisible delimiter markers we get from expanding
+                // an `expr`.
+                let len1 = "/*«*/ ".len();
+                let len2 = "/*»*/ ".len();
+                let val = &val[len1..val.len() - len2];
```

Comment:
https://github.com/petrochenkov/rust/commit/2510663457d01ecfb695c867499c51cd1cd64501

I wanted to submit it as a PR but then realized that it's hard to test without changes from this PR.
But we'll need to backport it beta anyway, because https://github.com/rust-lang/rust/pull/96682 is already on beta.

Reply:
Thanks, I have incorporated this change into my patch stack.